ddtrace/appsec/_remoteconfiguration.py

```diff
@@ -26,9 +33,46 @@ def enable_appsec_rc():
 
     if _appsec_rc_features_is_enabled():
         from ddtrace.internal.remoteconfig import RemoteConfig
+        from ddtrace.internal.remoteconfig.constants import ASM_DATA_PRODUCT
         from ddtrace.internal.remoteconfig.constants import ASM_FEATURES_PRODUCT
 
         RemoteConfig.register(ASM_FEATURES_PRODUCT, appsec_rc_reload_features(tracer))
+        RemoteConfig.register(ASM_DATA_PRODUCT, appsec_rc_reload_features(tracer))
+
+
+def _appsec_rules_data(tracer, features):
+    # type: (Tracer, Union[Literal[False], Mapping[str, Any]]) -> None
+    if features and tracer._appsec_processor:
+        rules_data = features.get("rules_data", [])
+        if rules_data:
+            log.debug("Reloading Appsec rules data: %s", rules_data)
```

Comment:
```suggestion
            log.debug("Reloading ASM rules data: %s", rules_data)
```
(if we're going with `ASM`, I see `ASM` being used below)